tgstation.dme

```diff
@@ -35,7 +35,6 @@
 #include "code\__DEFINES\hud.dm"
 #include "code\__DEFINES\inventory.dm"
 #include "code\__DEFINES\is_helpers.dm"
-#include "code\__DEFINES\language.dm"

```

Comment:
Its more than that, pulling from upstream removed several files but the rest of the code doesn't reflect that so im getting hundreds of errors.